ca/config.go

```diff
@@ -124,10 +124,15 @@ func SigningPolicy(certExpiry time.Duration) *cfconfig.Signing {
 		certExpiry = DefaultNodeCertExpiration
 	}
 
+	notBefore := time.Now().Round(time.Minute).Add(CertDefaultBackdate).UTC()
+	notAfter := time.Now().Round(time.Minute).Add(certExpiry).UTC()
+
 	return &cfconfig.Signing{
 		Default: &cfconfig.SigningProfile{
-			Usage:  []string{"signing", "key encipherment", "server auth", "client auth"},
-			Expiry: certExpiry,
+			Usage:     []string{"signing", "key encipherment", "server auth", "client auth"},
+			Expiry:    certExpiry,
+			NotBefore: notBefore,
+			NotAfter:  notAfter,
```

Comment:
CFSSL is silly and doesn't seem to allow me to not specific `Expiry`. Tells me this is an invalid profile. Didn't dig super far into it though, I might be missing something obvious.

This is the code that fills in the `notBefore` and `notAfter`: https://github.com/cloudflare/cfssl/blob/f8a8adaa9d48141125b78c23553a174845661522/signer/signer.go#L280